crates/cli/Cargo.toml

```diff
@@ -20,5 +20,8 @@ libloading = "0.8"
 cargo_metadata = "0.20"
 semver = "1.0"
 
+[target.'cfg(unix)'.dependencies]
```

Comment:
Using it to determine if current user is `root`/`sudo` or not! This crate is already in `Cargo.lock` and at least used by two other deps.